src/ui/CardOverlay/CardOverlay.ts

```diff
@@ -0,0 +1,94 @@
+import {Component} from '../Base/Component';
+import {ComponentOptions} from '../Base/ComponentOptions';
+import {IComponentBindings} from '../Base/ComponentBindings';
+import {Initialization} from '../Base/Initialization';
+import {$$} from '../../utils/Dom';
+import {Assert} from '../../misc/Assert';
+
+export interface ICardOverlayOptions {
+  title: string;
+  icon?: string;
+}
+
+/**
+ * This component is used to display a button which triggers an overlay on a
```

Comment:
I must admit I am not totally sure what this component does.

It's a component that wraps a singular button in a CardActionBar ?

Then, shouldn't it be named simply "CardAction" ?